app/models/comment.rb

```diff
@@ -1,2 +1,8 @@
 class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def comments_counter
+    comments.increment!(:comments_counter)
```

Comment:
[REQUIRED] Kindly switch this to `post.increment`. You are expected to update the comment counter for a post.